zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -1501,6 +1554,24 @@ components:
             associated task.
           type: object
           nullable: true
+    JobErrorRequest:
+      type: object
+      properties:
+        errorCode:
+          description: >
+            The error code that will be matched with an error catch event.
+          type: string
+        errorMessage:
+          description: >
+            An error message that provides additional context.
+          type: string
+          nullable: true
+        variables:
+          additionalProperties: true
+          description: >
+            JSON object that will instantiate the variables at the local scope of the error catch event that catches the thrown error.
+          type: object
+          nullable: true
```

🔧 We can mark the `errorCode` as required as well. I think this is the syntax (please double check):
```suggestion
          nullable: true
      required:
        - errorCode
```

---

Should we do the same also for `JobActivation` ?

---

Oh, yeah, good point 👍 I also just learned about it recently and didn't know it back then. I think the type, maxJobsToActivate, and timeout are required to be provided by the user. Any remaining numeric attributes can actually receive a default 0, I think. Feel free to make a proposal. Otherwise, we can also handle this in a follow-up issue.